src/Microsoft.SqlTools.ServiceLayer/TaskServices/SqlTask.cs

```diff
@@ -96,6 +96,13 @@ public SqlTask(TaskMetadata taskMetdata, Func<SqlTask, Task<TaskResult>> testToR
             });
         }
 
+        public void Run()
+        {
+            RunAsync().ContinueWith(task =>
```

Comment:
The reason is I wanted to add a Sync method because I was getting warnings when calling the Async method inside another async but was not waiting for it. this method runs the task doesn't wait for the result  

---
In reply to: [122304875](https://github.com/Microsoft/sqltoolsservice/pull/381#discussion_r122304875) [](ancestors = 122304875,122303762)